doc/stolonctl.md

```diff
@@ -1,18 +1,39 @@
 ## Stolon Client (stolonctl)
 
-`stolonctl` is the stolon client which controls the stolon cluster(s)
+`stolonctl` is the stolon client which controls the stolon cluster(s) and may be executed inside a pod running a stolon component or externally. 
 
-It needs to communicate with the cluster store (providing `--store-backend` and related option) on which the requested cluster name (`--cluster-name`) is running.
+Since `stolonctl`needs to communicate with the cluster store, it requires providing the requested cluster name (`--cluster-name`), its store backend (`--store-backend`), and when using "kubernetes" as store the kind of kubernetes resources (`--kube-resource-kind`).
 
-To avoid repeating for every command (or inside scripts) all the options you can export them as environment variables. Their name will be the same as the option name converted in uppercase, with `_` replacing `-` and prefixed with `STOLONCTL_`.
+Example, check stolon cluster status for a cluster named "blue-stolon" using "etcdv3" as a store:
+```
+$ stolonctl status --cluster-name=blue-stolon --store-backend=etcdv3 --store-endpoints=http://etcd-0.etcd:2379,http://etcd-1.etcd:2379,http://etcd-2.etcd:2379
+```
+
+To avoid repeating the argurments on every command (or inside scripts), all the options can be exported as environment variables. Their name will be the same as the option name converted in uppercase, with `_` replacing `-` and prefixed with `STOLONCTL_`.
 
-Ex.
+For example:
 ```
 STOLONCTL_STORE_BACKEND
 STOLONCTL_STORE_ENDPOINTS
 STOLONCTL_CLUSTER_NAME
 ```
 
+### Running in Kubernetes
+
+`stolonctl` behaves like [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) when choosing how to access the k8s API servers: 
+* When run inside a pod it uses the pod service account to connect to the k8s API servers.
+* When run externally it honors the $KUBECONFIG environment variable to connect. It is thus possible to use the default `~/.kube/config` file or an overriden kube-config file path, context and namespace to set the `stolonctl` options `--kubeconfig`, `--kube-context` and `--kube-namespace`.
+
+Example using one of the stolon proxy pods to run `stolonctl` from to check the status of a cluster named "red-stolon" using "kubernetes" as a store and "configmap" as the resources: 
+```
```

Comment:
There's also the option to execute a one-shot pod like explained in the k8s example:
https://github.com/sorintlab/stolon/blob/master/examples/kubernetes/README.md#initialize-the-cluster

I used this in the example since you don't need to find a pod.